### PR TITLE
Implemented method getStatistic

### PR DESCRIPTION
**Pull request by artyomwhite**

### CODE REVIEW

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+    private String readFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            return builder.toString();
+
```
Comment (Vasul-Pankiv) [inline src/main/java/core/basesyntax/WorkWithFile.java:28]: redundant empty line

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+    private String createReport(String fileContent) {
+        String[] lines = fileContent.split(System.lineSeparator());
+        int totalSupply = 0;
+        int totalBuy = 0;
+        for (String line : lines) {
+            String[] words = line.split(",");
+            String operatorType = words[OPERATOR_TYPE_INDEX];
+            int operatorSum = Integer.parseInt(words[OPERATOR_SUM_INDEX]);
+            if (operatorType.equals("supply")) {
```
Comment (Vasul-Pankiv) [inline src/main/java/core/basesyntax/WorkWithFile.java:42]: make operation name a constant value

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 3]
```diff
@@ -1,7 +1,61 @@
...
+            throw new RuntimeException("Can`t read file", e);
+        }
+    }
+
+    private String createReport(String fileContent) {
+        String[] lines = fileContent.split(System.lineSeparator());
+        int totalSupply = 0;
+        int totalBuy = 0;
+        for (String line : lines) {
+            String[] words = line.split(",");
```
Comment (Vasul-Pankiv) [inline src/main/java/core/basesyntax/WorkWithFile.java:39]: make separator a constant value

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]
Realy great solution!

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 4]
```diff
@@ -1,7 +1,60 @@
...
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            return builder.toString().trim();
+        } catch (IOException e) {
+            throw new RuntimeException("Не удалось прочитать файл", e);
```
Comment (kerrrusha) [inline src/main/java/core/basesyntax/WorkWithFile.java:31]: only english please

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 5]
```diff
@@ -1,7 +1,60 @@
...
+        }
+        return String.format(REPORT_TEMPLATE, OPERATION_SUPPLY, totalSupply, "buy", totalBuy,
+                "result", totalSupply - totalBuy);
+    }
 
+    private void writeReportToFile(String toFileName, String report) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Не удалось записать файл", e);
```
Comment (kerrrusha) [inline src/main/java/core/basesyntax/WorkWithFile.java:57]: only english please

File: src/main/java/core/basesyntax/WorkWithFile.java (hunk 3 above)
Comment (kerrrusha) [reply src/main/java/core/basesyntax/WorkWithFile.java:40]: not fixed

**Review by kerrrusha — CHANGES_REQUESTED** [review]
overall very good!

**Pull request closed without merging**